__tests__/seed.test.js

```diff
@@ -57,9 +58,19 @@ describe("GET /api", () => {
         .expect(200)
         .then(({body}) => {
             console.log({body})
-            expect(body.length > 0).toBe(true)
-            expect(body[0])
+            
+                expect.objectContaining({ description: expect.any(String),
+                queries: expect.any(Array),
+                exampleResponse: expect.any(Object)
+                
+                })
+                })
+                
+            })
         })
+    test("responds with 404 when incorrect spelling", () => {
```

Comment:
A bad path test is not related to any one endpoint and so should not live in a describe block for any endpoint. We don't access the controller or model for any other route when a bad path is accessed. So it only needs testing for once and separately.